.github/workflows/release.yml

```diff
@@ -91,7 +91,7 @@ jobs:
         if: success()
         uses: micronaut-projects/github-pages-deploy-action@master
         env:
-          BETA: ${{ contains(steps.release_version.outputs.release_version, 'M') || contains(steps.release_version.outputs.release_version, 'RC') }}
```

Comment:
why is the non-default branch a beta? we release patch versions of previous minors all the time

Reply:
oh this is from the commons repo? im sure theres a good reason then.

Reply:
No idea...  This comes from https://github.com/micronaut-projects/micronaut-crac/pull/175

Which is copying the template changes by @sdelamo https://github.com/micronaut-projects/micronaut-project-template/commit/48c581a704f38bbddb1acab4ae75e14e7e313bfd